src/spdx2/agent/spdx2.php

```diff
@@ -198,24 +198,24 @@ protected function getUri($fileBase,$packageName)
    * @param string[] $licenses
    * @return string
    */
-  protected function implodeLicenses($licenses)
+  protected function implodeLicenses($licenses, $prefix="")
   {
     sort($licenses);
     if(count($licenses) == 3 &&
        ($index = array_search("Dual-license",$licenses)) !== false)
     {
-      return $licenses[$index===0?1:0] . " or " . $licenses[$index===1?2:1];
+      return $prefix . $licenses[$index===0?1:0] . " OR " . $prefix . $licenses[$index===1?2:1];
```

Comment:
I do not think that they are necessary since they will be added on demand in line https://github.com/fossology/fossology/pull/666/files#diff-7bba19f12bab5e8b7e4b602c33f2e955R212